### ISSUE TITLE
How to get "register date" of account?

### ISSUE THREAD

**Issue by grisha2217** [adw0rd/instagrapi-rest#58]
Hello all. I need to get register date for account. How to do it?

**Comment by RoyalZebra** [comment]
Yes, please let us know. this is a feature which is present in hikerAPI but not in this project. can anyone help?

**Comment by RoyalZebra** [comment]
> Send your PR for this repo, we will accept it
> 
> But we recommend using https://hikerapi.com/p/5GBWznd3

You want me to send a PR? but i cannot contribute? basically I want you to add the feature which is available to hikerAPI but is absent in this repo

**Comment by I-NRL** [comment]
> Hello all. I need to get register date for account. How to do it?

Ig bulk account creater tool available, dm me for more info,

Custom proxy,
Temp mail,
Web api
Follow/unfollow 
Bio, profile icon change,

auto comment, like, unlike and more
